website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -95,4 +140,48 @@ The `content` supports any custom content, local components, or Helios component
 
 ### Custom
```

Comment:
```suggestion
#### Custom
```

Reply:
The "Default" headline needs to be adjusted as well but I can't comment on it because there isn't a diff